api/app/signals/apps/api/v1/serializers/routing.py

```diff
@@ -0,0 +1,18 @@
+from rest_framework import serializers
+
+from signals.apps.api.generics.permissions import SIAPermissions
+from signals.apps.api.v1.serializers.nested import _NestedDepartmentModelSerializer
+from signals.apps.signals.models import RoutingExpression
+
+
+class RoutingDepartmentSerializer(serializers.ModelSerializer):
+    department = _NestedDepartmentModelSerializer(
+        source='_department',
+        many=False,
+        required=False,
+        permission_classes=(SIAPermissions,),
```

Comment:
We may need a specific permission for this, but let's save that for later (other PR).

Reply:
check, agreed